editor/header/index.js

```diff
@@ -1,12 +1,23 @@
 /**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
  * Internal dependencies
  */
 import './style.scss';
+import MultiSelectHeader from './multi-select';
 import ModeSwitcher from './mode-switcher';
 import SavedState from './saved-state';
 import Tools from './tools';
+import { getSelectedBlocks } from '../selectors';
+
+function Header( { selectedBlocks } ) {
+	if ( selectedBlocks.length ) {
```

Comment:
If my confusion could serve as evidence, it would be good if we could better distinguish or eliminate the differences between the separate selected state.